beacon-chain/verification/blob.go

```diff
@@ -45,6 +45,22 @@ var GossipSidecarRequirements = []Requirement{
 	RequireSidecarProposerExpected,
 }
 
+// GossipSidecarRequirements defines the set of requirements that BlobSidecars received on gossip
+// must satisfy in order to upgrade an ROBlob to a VerifiedROBlob.
+var SpectestSidecarRequirements = []Requirement{
+	RequireBlobIndexInBounds,
+	RequireNotFromFutureSlot,
+	RequireSlotAboveFinalized,
+	RequireValidProposerSignature,
+	RequireSidecarParentSlotLower,
+	//RequireSidecarParentSeen,
+	//RequireSidecarParentValid,
```

Comment:
Did you mean to comment these out? 

If so, consider adding some method like 

```go
var SpectestSidecarRequirements = xor(GossipSidecarRequirements, []Requirement{
  // Requiring parent checks in spectest is not possible.
  RequireSidecarParentSeen, 
  RequireSidecarParentValid,
})
```

This way we won't have to ensure two slices are updated in the future.

Reply:
implemented this, please take a look!

fwiw the main downside I see is that it makes it a little hard to make out what requirements you're keeping for lists where you're excluding most things, like initial-sync/backfill. but otherwise I'm good with it if you are.